### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by garyb**
A bunch of things @kRITZCREEK and I encountered while working on SQL generation that could be tweaked for a better experience.

### CODE REVIEW

File: src/SqlSquared/Parser.purs [hunk 1]
```diff
@@ -413,17 +414,17 @@ import_ = asErrorMessage "import declaration" do
   path ← Pt.parseAnyDirPath P.fail s
   pure $ Sig.Import path
 
-variable ∷ ∀ m t. SqlParser' m t
```
Comment (kritzcreek) [inline src/SqlSquared/Parser.purs]: Was this change intended?

**Review by kritzcreek — CHANGES_REQUESTED** [review]
Looks good, just seems like one typo

**Comment by garyb** [comment]
Oops! Should be good now.

**Review by kritzcreek — APPROVED** [review]

File: src/SqlSquared/Signature.purs [hunk 2]
```diff
@@ -528,11 +528,11 @@ genFunctionDecl n = do
 genImport ∷ ∀ m a. Gen.MonadGen m ⇒ MonadRec m ⇒ m (SqlDeclF a)
 genImport = map Import Pt.genAnyDirPath
 
-genIdent ∷ ∀ m. Gen.MonadGen m ⇒ m String
+genIdent ∷ ∀ m. Gen.MonadGen m ⇒ m ID.Ident
```
Comment (cryogenian) [inline src/SqlSquared/Signature.purs]: This should live in `SqlSquared.Signature.Ident` I think.

**Review by cryogenian — COMMENTED** [review]
Non-blocking but good to have comment about moving `genIdent` :)

File: src/SqlSquared/Signature.purs (hunk 2 above)
Comment (garyb) [reply src/SqlSquared/Signature.purs]: Makes sense 👍

**Review by cryogenian — APPROVED** [review]
Thanks!
